src/users/users.controller.spec.ts

```diff
@@ -140,21 +140,12 @@ describe('Users Controller', () => {
     });
 
     describe('deleteUser() method', () => {
-      it('should call method deleteUser in userService', () => {
+      it('should call method deleteUser in userService', async () => {
         const createSpy = jest.spyOn(usersService, 'deleteUser');
 
-        usersController.deleteUser('anyid');
+        await usersController.deleteUser('anyid');
         expect(createSpy).toHaveBeenCalledWith('anyid');
       });
-
-      it('should throw an exception if it not find a user', () => {
-        jest
-          .spyOn(usersService, 'deleteUser')
-          .mockRejectedValueOnce(new NotFoundException());
-        expect(usersController.deleteUser('anyid')).rejects.toThrow(
-          new NotFoundException(),
-        );
-      });
```

Comment:
user controller just forwards on whatever user service does. There's no `try/catch` so no real need to assert that an error from the service propagates onward